{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/settings/base.py

```diff
@@ -1,8 +1,14 @@
 # Django settings for project project.
-from os import environ
-from django.core.exceptions import ImproperlyConfigured
+from environ import Env, Path
 import pathlib
+from django.core.exceptions import ImproperlyConfigured
+
+env = Env()
```

Comment:
I don't think we are putting `.env` on heroku so maybe this call should just be in `local.py`? Need to think about the organization some.